converter.go

```diff
@@ -610,6 +646,33 @@ func arrowToValue(
 			}
 		}
 		return err
+	case vectorType:
+		vectorData := srcValue.(*array.FixedSizeList)
+		datatype := vectorData.DataType().(*arrow.FixedSizeListType)
```

Comment:
Does this work only for arrow? What should happen if a client uses `ALTER SESSION SET GO_QUERY_RESULT_FORMAT = JSON`?